src/nvim/generators/gen_unicode_tables.lua

```diff
@@ -252,7 +252,8 @@ local build_emoji_table = function(ut_fp, emojiprops, doublewidth, ambiwidth)
 
       -- Characters below 1F000 may be considered single width traditionally,
       -- making them double width causes problems.
```

Comment:
```suggestion
      -- Characters below 1F000 may be considered single width traditionally.
      -- And Regional Indicator Symbols 0x1F1E6-0X1F1FF are almost always used
      -- in pairs to represent a country flag, so we special-case them as
      -- single-width to avoid 4 virtual columns. #16447
```